src/components/Attachment/index.tsx

```diff
@@ -162,12 +162,13 @@ export default function Attachment(props) {
         pConn.getContextName()
       )
       .then((fileRes) => {
-        pConn.attachmentsInfo = {
+        const reqObj = {
           type: "File",
           attachmentFieldName: valueRef,
           category: categoryName,
           ID: fileRes.ID
         };
+        pConn.attachmentsInfo = PCore.getPCoreVersion()?.includes('8.8') ? [reqObj] : reqObj;
```

Comment:
I think you should write the logic same as other sdks like condition to support 8.7 and above versions
`PCore$.getPCoreVersion()?.includes('8.7') ? reqObj : [reqObj];`